### PR TITLE
More debugging and error checking

### PR DESCRIPTION
**Pull request by adybbroe**
- [ ] Closes #xxxx 
 - [x] Tests added 
 - [x] Tests passed: Passes ``pytest`` 
 - [x] Passes ``flake8`` 
 - [ ] Fully documented

### CODE REVIEW

File: activefires_pp/post_processing.py [hunk 1]
```diff
@@ -571,6 +576,11 @@ def _generate_no_fires_messages(self, input_msg, msg_string):
 
         return publish_messages
 
+    def _check_boarders_shapes_exists(self):
```
Comment (mraspaud) [inline activefires_pp/post_processing.py:579]: typo: borders, not boarders

File: activefires_pp/tests/conftest.py [hunk 2]
```diff
@@ -171,3 +171,13 @@ def fake_past_detections_dir(tmp_path):
         fpt.write(PAST_ALARMS_MONSTERAS2)
 
     yield file_path.parent
+
+
+@pytest.fixture
+def fake_national_boarders_shapefile(tmp_path):
```
Comment (mraspaud) [inline activefires_pp/tests/conftest.py:177]: typo

File: activefires_pp/tests/test_fires_filtering.py [hunk 3]
```diff
@@ -278,10 +281,50 @@ def test_general_national_fires_filtering(get_global_mask, setup_comm, get_confi
...
     assert isinstance(result, pd.core.frame.DataFrame)
     assert len(result) == 1
     np.testing.assert_almost_equal(result.iloc[0]['latitude'], 59.52483368)
     np.testing.assert_almost_equal(result.iloc[0]['longitude'], 17.1681633)
     assert outmsg == ["my fake output message"]
+
+
+@pytest.mark.usefixtures("fake_national_boarders_shapefile")
+@patch('socket.gethostname')
+@patch('activefires_pp.post_processing.read_config')
```
Comment (mraspaud) [inline activefires_pp/tests/test_fires_filtering.py:295]: why does `read_config` need to be patched? could we use a minimal configuration files here instead?

**Review by mraspaud — COMMENTED** [review]
Looks good with more debug messages. A couple of typos and a question inline.

File: activefires_pp/tests/test_fires_filtering.py (hunk 3 above)
Comment (adybbroe) [reply activefires_pp/tests/test_fires_filtering.py]: Yes, right, fixed

**Pull request merged**